docker/run

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+import argparse
+import subprocess
+from subprocess import PIPE
+import os
+
+KNOWN_NVIDIA_ERRORS = [
+    ("Failed to initialize NVML: Driver/library version mismatch"),
+    ("NVIDIA-SMI has failed because it couldn't communicate with the NVIDIA driver. "
+     "Make sure that the latest NVIDIA driver is installed and running."),
+]
+
+
+def is_nvidia():
+    try:
+        everything, _ = subprocess.Popen(["nvidia-smi", "--query-gpu=driver_version", "--format=noheader,csv"],
+                                         stdout=PIPE, stderr=PIPE).communicate()
+        everything = everything.strip()
+
+        if everything in KNOWN_NVIDIA_ERRORS:
+            return False
+        
+        return True
+    except OSError:
+        return False
+    
+    return False
+
+
+def run_command(command):
+    subprocess.call(command, shell=True)
+
+
+def get_repo_root():
+    return subprocess.check_output('git rev-parse --show-toplevel'.split()).strip()
+
+
+def run_dev_environment(command):
+    user = os.getlogin()
+    uid = os.getuid()
+    docker_args = []
+    image_name = 'create_ros_kinetic_gazebo9'
+
+    docker_args.append("-it")
```

Comment:
Have you ever used [this](https://docker-py.readthedocs.io/en/stable/)?

Reply:
So far I had never encountered that. Maybe it's even easier to handler docker from python like that